experiments/jz/joint_training_toy/joint_training_toy1/04_do_training.slurm

```diff
@@ -45,16 +46,12 @@ while [[ $string ]]; do
 done
 REPO_DIR="${DATASETS_CUSTOM}/${myarray[-1]}"
 
-python bsmetadata/train.py \ # (change me and remove the comment! if you have a specific script)
+python bsmetadata/train.py \
     data_config.experiment="with_metadata" \
-    data_config.metadata_config.metadata_list=["html"] \
+    data_config.metadata_config.metadata_list=["entity", "timestamp", "url", "website_description"] \
     data_config.metadata_config.max_seq_len=1024 \
     data_config.dataset_name="${REPO_DIR}" \
-    data_config.train_file="XXX" \ # (change me and remove the comment! e.g "nq-train-0\[0-2\].jsonl.gz" or remove arg)
-    data_config.validation_file"XXX" \ # (change me and remove the comment! e.g "nq-dev-00.jsonl.gz" or remove arg)
     data_config.preprocessing_num_workers=80 \
-    data_config.per_device_eval_batch_size=3 \
-    data_config.per_device_train_batch_size=3 \
     out_dir="${SCRATCH}/metadata_outputs/${SLURM_JOB_ID}" \
     jobid="${SLURM_JOB_ID}" \
     do_train=True \
```

Comment:
As we didn't specify a file regex for the `validation_file`, I think we should put `do_eval=False` (or specify a file for the evaluation)

Reply:
By default, it will do a 95-5 train-val split. I don't think it will be a problem. (I test ran it.)